pkg/operator/edgefs/cluster/controller.go

```diff
@@ -384,20 +362,19 @@ func (c *ClusterController) handleDelete(clust *edgefsv1beta1.Cluster, retryInte
 	return nil
 }
 
-func (c *ClusterController) updateClusterStatus(namespace, name string, state edgefsv1beta1.ClusterState, message string) error {
+func (c *ClusterController) updateClusterStatus(namespace, name string, state edgefsv1beta1.ClusterState, message string) {
```

Comment:
Changing edgefs code here seems out of scope for a Ceph PR.

Reply:
This status change method is the same for both ceph and edgefs. When there is a pattern used across multiple backends that needs updating I'd rather update it at the same time. There is at least a separate commit for this change. @dyusupov Does that work for you?

Reply:
With separate commit and @dyusupov 's ok, seems okay to me.

Reply:
For clarify i decided to split it out into #3509. I'll rebase this after that merges.